packages/wonder-blocks-tooltip/components/tooltip-content.js

```diff
@@ -53,12 +54,23 @@ export default class TooltipContent extends React.Component<Props> {
     render() {
         const title = this._renderTitle();
         const children = this._renderChildren();
+        const containerStyle = title ? styles.withTitle : styles.withoutTitle;
         return (
-            <View>
+            <View style={containerStyle}>
                 {title}
                 {title && children && <Strut size={Spacing.xxxSmall} />}
                 {children}
             </View>
         );
     }
 }
+
+const styles = StyleSheet.create({
+    withoutTitle: {
+        margin: "10px 16px",
+    },
+
+    withTitle: {
+        margin: 16,
```

Comment:
We should update the 16pxs to Spacing.medium -- I'm not sure why we're using 10px, it's not one of our standard spacing units!

Reply:
That's what the design says. I didn't use `Spacing.medium` because the 10px was a spacing option and it seemed better to be consistent with that.